python/xgbserver/__main__.py

```diff
@@ -1,12 +1,12 @@
 import kfserving
 import argparse
 
-from model import XGBoostModel
+from xgbserver import XGBoostModel
 
 DEFAULT_MODEL_NAME = "default"
 DEFAULT_LOCAL_MODEL_DIR = "/tmp/model"
 
-parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(parents=[kfserving.server.parser])
```

Comment:
I was getting the inheritance behavior without parents. I think this is fine, but I'm curious what issue you were running into.

Reply:
Running from the command line I wasn't getting the inheritance.

Reply:
Very strange -- I wonder if this is due to versioning. I think it's fine to be explicit.

Reply:
which versioning? I ,eam when I run:

```
python -m xgbserver
usage: __main__.py [-h] [--http_port HTTP_PORT] [--grpc_port GRPC_PORT]
                   [--protocol {tensorflow.http,seldon.http}] --model_dir
                   MODEL_DIR [--model_name MODEL_NAME]
__main__.py: error: the following arguments are required: --model_dir
```

Reply:
That's expected and what I get too. Just point to a model file somewhere. I have some samples in gs://kfserving-samples that we can wire up in a README. I won't be getting much work done this weekend  other than reviewing though.

Reply:
@ellis-bigelow it maybe also be good to put strawman dev instructions out there now...